libs/blocks/how-to/how-to.js

```diff
@@ -52,6 +53,7 @@ const getHowToInfo = (el) => {
   if (!infoDiv) return {};
 
   const heading = infoDiv.firstElementChild;
+  heading.classList.add('heading-l');
```

Comment:
Interesting. Does this make all headings have a baseline of `heading-l`'s for the block unless overwritten otherwise? Seems like a good default, but kind of odd for authoring since there would not be a difference between h2,h3,h4. But based on the block it seems it is built that way anyway.